soh/Makefile

```diff
@@ -24,18 +24,18 @@ WARN := \
 	-funsigned-char \
 	-fno-stack-protector -fno-common -fno-zero-initialized-in-bss -fno-strict-aliasing -fno-inline-functions -fno-inline-small-functions -ffreestanding -fwrapv \
 
-CXXFLAGS := $(WARN) -std=c++20 -D_GNU_SOURCE -fpermissive -no-pie -nostdlib
-CFLAGS := $(WARN) -std=c99 -D_GNU_SOURCE -no-pie -nostdlib
+CXXFLAGS := $(WARN) -std=c++20 -D_GNU_SOURCE -fpermissive -no-pie -nostdlib $(shell pkg-config --cflags sdl2 glew) $(shell sdl2-config --cflags)
+CFLAGS := $(WARN) -std=c99 -D_GNU_SOURCE -no-pie -nostdlib $(shell pkg-config --cflags sdl2 glew) $(shell sdl2-config --cflags)
```

Comment:
don't think we need both `pkg-config --cflags sdl2` and `sdl2-config --cflags`, just the `sdl2-config` one should be enough

they output equivalent things on my (mac) machine 

```sh
jeff@winters ~ $ pkg-config --cflags sdl2
-D_THREAD_SAFE -I/opt/local/include/SDL2
jeff@winters ~ $ sdl2-config --cflags
-I/opt/local/include/SDL2 -D_THREAD_SAFE
```

Reply:
same on LInux! I'll update this